pkg/addons/data/Deployment.apps/default/router.yaml

```diff
@@ -62,26 +52,23 @@ spec:
         imagePullPolicy: Always
         livenessProbe:
           httpGet:
-            host: localhost
             path: /healthz
             port: 1936
             scheme: HTTP
           initialDelaySeconds: 10
+        readinessProbe:
```

Comment:
might as well move this back down because it'll just move itself when we next run tools/update-addons.go.